docs/core/preview3/tools/dotnet-add-package.md

```diff
@@ -17,20 +17,20 @@ ms.assetid: 88e0da69-a5ea-46cc-8b46-5493242b7af9
 
 ## Name
 
-`dotnet-add package`
+`dotnet-add package` - Adds a package reference to a project file.
 
 ## Synopsis
 
 ```
-dotnet add [<PROJECT>] package <PACKAGE_NAME> [-v|--version] [-f|--framework] [-s|--source] [--package-directory] [-h|--help]
+dotnet add [<PROJECT>] package <PACKAGE_NAME> [-v|--version] [-f|--framework] [-s|--source] [--package-directory]
```

Comment:
Removing `[-h|--help]` from this line infers that you cannot use the option with `package` or `dotnet add` which you can. I suggest leaving this.